### PR TITLE
Allow customization of aws-secret name and keys in Helm Chart

### LINKED ISSUE #1492: Helm: Allow to set custom secret name & keys
**Is your feature request related to a problem?/Why is this needed**
We use helm to install the EBS driver on our cluster. However the default secret name `aws-secret` is so generic and can't be changed atm, which makes it hard to use on a larger cluster. Would be nice if one could change the name of the secret and its keys.

/feature

**Describe the solution you'd like in detail**
```
            - name: AWS_ACCESS_KEY_ID
              valueFrom:
                secretKeyRef:
                  name: {{ .Values.controller.aws_access_key.secret_name }}
                  key: {{ .Values.controller.aws_access_key.key_for_id }}
                  optional: true
            - name: AWS_SECRET_ACCESS_KEY
              valueFrom:
                secretKeyRef:
                  name: {{ .Values.controller.aws_access_key.secret_name }}
                  key: {{ .Values.controller.aws_access_key.key_for_secret }}
                  optional: true
```

### PR DESCRIPTION
**Pull request by AndrewSirenko**
**Is this a bug fix or adding new feature?**

Feature 

**What is this PR about? / Why do we need it?**

Closes #1492; supersedes #1571 

Currently, users that grant the driver IAM permissions with a secret object are only allowed to use the secret name `aws-secret`, and key names `key_id` and `access_key` due to [charts/aws-ebs-csi-driver/templates/controller.yaml](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/2bac0f86b8fe199aeaada4786024923f27374a39/charts/aws-ebs-csi-driver/templates/controller.yaml#L107-L118) not templating those parameters. 

This PR would allow users to customize their secret and key names via the helm parameters `awsAccessSecret.name`, `awsAccessSecret.keyId`, and `awsAccessSecret.accessKey` in [this PR's values.yaml](https://github.com/AndrewSirenko/aws-ebs-csi-driver/blob/be429bef2deb3371e5abb022f2ab03d525453eab/charts/aws-ebs-csi-driver/values.yaml#L148-L151).

**What testing is done?** 

Granted CSI-driver IAM permissions using a secret object with custom secret, Key ID, and Access Key names. Manually installed CSI-Driver via helm with changed default values, leading to the following CSI Driver controller deployment description (see screenshot).  

CSI Driver Controlley Deployment Evidence

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ There are several methods to grant the driver IAM permissions:
...
 Therefore, if you are using k8s 1.24 or higher, you need to set `true` to `controller.serviceAccount.autoMountServiceAccountToken`.
-* Using secret object - create an IAM user, attach the policy to it, then create a generic secret called `aws-secret` in the `kube-system` namespace with the user's credentials
+* Using secret object - create an IAM user, attach the policy to it, then create a generic secret in the `kube-system` namespace with the user's credentials. The snippet below creates the generic secret named `aws-secret` that the driver accepts by default. You can customize this default name via Helm by setting `Value.awsAccessSecret.name` to a custom secret name. Additionally, you may use an existing secret with an AWS access key by configuring `Value.awsAccessSecret.name` with the name of your existing secret, `Value.awsAccessSecret.keyId` with the key of the AWS_ACCESS_KEY_ID, and `Value.awsAccessSecret.accessKey` with the key of the AWS_SECRET_ACCESS_KEY.
```
Comment (ConnorJC3) [inline docs/install.md:34]: configuring `Value.awsAccessSecret.name` -> configuring the Helm parameter `awsAccessSecret.name`

`Value.awsAccessSecret.keyId` -> `awsAccessSecret.keyId`

`Value.awsAccessSecret.accessKey` -> `awsAccessSecret.accessKey`

**Review by ConnorJC3 — CHANGES_REQUESTED** [review]

**Review by torredil — APPROVED** [review]
Thanks @AndrewSirenko /lgtm

**Review by ConnorJC3 — APPROVED** [review]
lgtm

**Pull request merged**